README.md

```diff
@@ -326,15 +326,19 @@ derive newtype instance decodeJsonAppUser :: DecodeJson AppUser
 If your data type has an instance of `Generic`, then you can use [purescript-argonaut-generic](https://github.com/purescript-contrib/purescript-argonaut-generic) to leverage `gEncodeJson` and `gDecodeJson` to write your instances:
```

Comment:
This would need a change, too, as it refers to `gEncodeJson`

Reply:
done, tnx